scripts/build.sh

```diff
@@ -164,6 +164,17 @@ cortxfs_make() {
         exit 1;
     fi
 
+    if [ -f "$CORTXFS_SRC/management/REST_doc.html" ]; then
+	    echo "Deleting Swagger Documentation"
+	    rm "$CORTXFS_SRC/management/REST_doc.html"
+    fi
+
+    sudo pip3 install PyYaml
```

Comment:
Question: Is this a good idea to install a package in build process? Usually we install all the packages required before installing cortx rpms.